topics/compute/openstack/docs/getting-started/creating-first-infrastructure.md

```diff
@@ -59,8 +59,40 @@ __5.__ Wait to be redirected back to our dashboard.
 All virtual machine instances running in the cloud have to be accessed remotely. The most common way of accessing
 an instance remotely is SSH. Using SSH requires a pair of keys - a public key and a private key.
 
+=== "CLI"
+
+    !!! note
+
+        Prefer using this method if possible. It is the most secure one, since your private key is kept
+        on your local machine at all times.
+
+
+    You can use the **ssh-keygen** command to create a new pair of private and public keys:
```

Comment:
pls add there package name ssh-keygen comes from (for ubuntu / rhel ) for instance
`You can use the **ssh-keygen** command (comming from ubuntu package  openssh-client) to create a new pair ...`